timemachine/cpp/src/nonbonded_common.cuh

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
why is this a `.cuh` file?

Reply:
Done in https://github.com/proteneer/timemachine/pull/1053/commits/0e617ed927eb592767a67f75c127ca6b322b6af0